external_resources/tasks.py

```diff
@@ -58,6 +61,10 @@ def check_external_resources(resources: list[int]):
             log.debug(ex)
             state.status = ExternalResourceState.Status.CHECK_FAILED
         else:
+            # Update the metadata of the resource with the status codes
+            resource.metadata[METADATA_URL_STATUS_CODE] = url_status
+            resource.metadata[METADATA_BACKUP_URL_STATUS_CODE] = backup_url_status
```

Comment:
@pt2302 We have these status codes already stored in `WebsiteContent.external_resource_state.external_url_response_code` and `WebsiteContent.external_resource_state.backup_url_response_code`. Do we have a use case planned to duplicate these to `WebsiteContent.metadata`?